File: CustomerOnBoardingBackend/CustomerOnboardingApplication/Models/Company.cs
using System;
using System.Collections.Generic;

namespace CustomerOnboardingApplication.Models;

public partial class Company
{
    public int CompanyId { get; set; }

    public string? NameOfCompany { get; set; }

    public string? TypeOfEntity { get; set; }

    public string? BusinessActivity { get; set; }

    public string? License { get; set; }

    public string? RegistrationNumber { get; set; }

    public DateTime? DateOfIncorporation { get; set; }

    public int? CountryId { get; set; }

    public int? CustomerId { get; set; }

    public virtual Country? Country { get; set; }

    public virtual Customer? Customer { get; set; }
}

File: CustomerOnBoardingApplicationFrontend/CustomerOnboardingApplication/src/Pages/Customer/Types/Company.ts
export interface Company{
    companyId: number;
    nameOfCompany: string;
    countryName: string
    typeOfEntity: string;
    businessActivity: "Banking" | "Fishing" | "Manufacturing";
    license: string | null;
    registrationNumber: string;
    dateOfIncorporation: Date;
}